docs/getting-started/install-crossplane.md

```diff
@@ -0,0 +1,132 @@
+---
+toc_hide: true
+---
+
+## Install Crossplane
+
+Crossplane installs into an existing Kubernetes cluster. 
+
+Install Crossplane either as an upstream open source distribution or from a [CNCF certified](https://github.com/cncf/crossplane-conformance) vendor distribution.
+
+The Crossplane community supports the upstream distribution.  
+The Crossplane vendor supports their specific Crossplane distribution. 
+
+{{< hint type="tip" >}}
+If you don't have a Kubernetes cluster create one locally with [minikube](https://minikube.sigs.k8s.io/docs/start/) or [kind](https://kind.sigs.k8s.io/).
+{{< /hint >}}
+
+{{< tabs "distros" >}}
+
+{{< tab "Community Supported" >}}
+### Install the Crossplane Helm chart
+
+Helm enables Crossplane to install all its Kubernetes components through a single _Helm Chart_.
+
+Enable the Crossplane Helm Chart repository:
+
+```shell
+helm repo add \
+crossplane-stable https://charts.crossplane.io/stable
+helm repo update
+```
+
+Helm supports a `dry-run` to see the changes Helm makes. Run the Helm dry-run to see all the Crossplane components.
+
+```shell
+helm install crossplane \
+--dry-run --debug \
+--namespace crossplane-system \
+crossplane-stable/crossplane
+```
+
+Last, install the Crossplane components using `helm install`.
+
+```shell
+helm install crossplane \
+--namespace crossplane-system \
+crossplane-stable/crossplane
+
+NAME: crossplane
+LAST DEPLOYED: Sun Oct 16 19:11:27 2022
+NAMESPACE: crossplane-system
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+Release: crossplane
+
+Chart Name: crossplane
+Chart Description: Crossplane is an open source Kubernetes add-on that enables platform teams to assemble infrastructure from multiple vendors, and expose higher level self-service APIs for application teams to consume.
+Chart Version: 1.9.1
+Chart Application Version: 1.9.1
+
+Kube Version: v1.25.2
+```
+
+Verify Crossplane installed with `kubectl get pods`.
+
+```shell
+kubectl get pods -n crossplane-system                                                                  
+NAME                                      READY   STATUS    RESTARTS   AGE
+crossplane-d4cd8d784-ldcgb                1/1     Running   0          54s
+crossplane-rbac-manager-84769b574-6mw6f   1/1     Running   0          54s
+```
+
+{{< /tab >}}
+
+{{< tab "Vendor distributions" >}}
+
```

Comment:
I would go the other way and say we should remove it from the other guide and centralize it into a section about other vendors or providers being added to the docs. This would tie into some of the changes in #3398

Reply:
> I would go the other way and say we should remove it from the other guide and centralize it into a section about other vendors or providers being added to the docs.

I'm indifferent as long as it is present and consistent.